config/vufind/config.ini

```diff
@@ -746,6 +749,9 @@ database          = mysql://root@localhost/vufind
 ;college = HTTP_COLLEGE
 ;major = HTTP_MAJOR
 ;home_library = HTTP_HOME_LIBRARY
+; You can here override mapping or required attributes for specific IdP here - name of configuration
+; file without .ini prefix
+;override = Shibboleth
```

Comment:
Is there an actual reason the ini file needs to be configurable? Would it suffice to have something like allow_configuration_override=true/false? Or does there need to be a setting at all? Why isn't Shibboleth.ini always read?

Reply:
I think this is a matter of backward compatibility. If you only have one IdP, you don't need Shibboleth.ini because the [Shibboleth] section of config.ini handles everything. You only need to load the .ini file if you have multiple IdPs to differentiate between. I do agree that at present, there doesn't seem to be a compelling reason to make the actual filename configurable, since it looks like all of the Shib-related settings will always be in a single file anyway. I'd be in favor of simplifying this to a boolean. I also wouldn't object to moving everything into that file in order to reduce the size of config.ini, but if we did that, we'd have to do more extra for backward compatibility/migration, so perhaps that is better addressed in a future PR.